oshi-core/src/main/java/oshi/util/ParseUtil.java

```diff
@@ -83,6 +83,10 @@
 
     private static final Map<String, Long> multipliers;
 
+    public static final Pattern whitespaceCloneWhitespace = Pattern.compile("\\s+:\\s");

```

Comment:
All the split remove the regex compile on each split. 
also, the replace of String also compile regex.